samples/tfm/tfm_hello_world/README.rst

```diff
@@ -27,12 +27,21 @@ Building and Running
 
 .. include:: /includes/build_and_run.txt
 
-Sample Output
-=============
+Testing
+=======
+
+After programming the sample, the following output is displayed in the console:
```

Comment:
OK for now, but should be cleaned up later.
Is it actually possible to build the sample stand-alone?

Reply:
If you remove the CONFIG_BUILD_WITH_TFM=y, it gives a compile error.